cmd/infra/aws/util/util.go

```diff
@@ -20,9 +32,8 @@ func NewSession(agent string) *session.Session {
 }
 
 // NewAWSRoute53Config generates an AWS config with slightly different Retryer timings
-func NewAWSRoute53Config(credentialsFile string, credKey string, credSecretKey string) *aws.Config {
-
-	awsRoute53Config := NewConfig(credentialsFile, credKey, credSecretKey, "us-east-1")
```

Comment:
@sjenning why was the region hardcoded here and is it okay to remove that?

Reply:
https://docs.aws.amazon.com/general/latest/gr/r53.html

> Route 53 in AWS Regions other than the Beijing and Ningxia Regions: specify us-east-1 as the Region.

Reply:
But it also says:

>  you can either leave the Region and endpoint unspecified, or specify the applicable Region:

And from what I can tell, things work fine? It maybe works because the `Region` setting in the `session` doesn't end up being used for global services?

Reply:
as long as it works, i'm good with either